core/loadpoint.go

```diff
@@ -116,7 +113,6 @@ type Loadpoint struct {
 	MeterRef        string `mapstructure:"meter"`    // Charge meter reference
 	Soc             SocConfig
 	Enable, Disable ThresholdConfig
-	GuardDuration   time.Duration // charger enable/disable minimum holding time
```

Comment:
Das gibt böse Fehler beim Start- siehe anderer PR.

Reply:
Verstehe ich nicht. Warum kompiliert das dann?
Was tun - wieder reinmachen oder das dort fixen wo auf dieses obsolete Member zugegriffen wird?

Reply:
Es geht um die Userkonfiguration, nicht ums compilieren. Die lädt dann nämlich nicht mehr.

Reply:
BC in die Release Notes schreiben? Wäre nicht das erste Mal.

Reply:
Nein, einfach nicht kaputt machen.

Reply:
Diese Anmerkung ist wenig hilfreich. Zu einer Grundsatzdiskussion gehören auch immer (mindestens) zwei. 
Das hier ist aber zumindest von meiner Seite aus keine, ich habe schlicht nicht verstanden wo das Problem liegt (und verstehe es immer noch nicht). Ist mir aber tatsächlich auch völlig egal.

Reply:
> Diese Anmerkung ist wenig hilfreich. 
> Ist mir aber tatsächlich auch völlig egal.

Und das ist hilfreich? Motivationssteigernd? Zusammenarbeitsfördernd?

>  ich habe schlicht nicht verstanden wo das Problem liegt 

Deshalb schrieb ich: 

> den Wert muss es in geeigneter Form noch geben, siehe anderer PR.

Hast Du denn mal rein geschaut? Da war das bereits gelöst.

Reply:
> > Diese Anmerkung ist wenig hilfreich.
> > Ist mir aber tatsächlich auch völlig egal.
> 
> Und das ist hilfreich? Motivationssteigernd? Zusammenarbeitsfördernd?

Du unterschlägst, dass zwischen den beiden Zeilen noch Text war.
Willst Du mir hier mit Gewalt Patzigkeit unterstellen?

Über was beklagst Du Dich ?

Mir ist in Anbetracht der daraus resultierenden Diskussion völlig egal, ob man GuardDuration  jetzt drin lässt oder nicht und warum hier kein BC gewollt ist und an anderer Stelle schon. Warum musss ich mich dafür rechtfertigen? Wäre es Dir lieber wenn ich dazu eine Meinung hätte?

> > ich habe schlicht nicht verstanden wo das Problem liegt
> 
> Deshalb schrieb ich:
> 
> > den Wert muss es in geeigneter Form noch geben, siehe anderer PR.
> 
> Hast Du denn mal rein geschaut? Da war das bereits gelöst.

Vielleicht bin ich ja ein bisschen begriffsstutzig - bitte entschuldige meine Beschränktheit!

Du solltest Dich mal fragen, wie meine Motivation ausschaut mich hier zu beteiligen.
Ich habe mich in das Thema involviert weil Du mich darum gebeten hast.

Ich erwarte keinen Dank. Aber dass ich als Dankeschön eins übergebraten bekomme - dazu noch mit einem verfälschten Zitat - finde ich keine angemessene Reaktion!

P.S: Du hast gesehen, dass ich die von Dir gewünschte Änderung schon lange vorgenommen habe?

Reply:
Bzw. jetzt geändert, nachdem ich es nochmals nachgesehen habe.

Reply:
@andig: Nochmals zur Erklärung - ich hatte wirklich nicht verstanden, dass und wie ein Mapping auf die Userkonfiguration erfolgt und wie der Umgang mit "deprecated" Konfigurationsparametern erfolgt. Und weil ich auf dem Stand vor https://github.com/evcc-io/evcc/pull/12053 aufgesetzt habe, habe ich auch nichts mutwillig "kaputt gemacht" was "bereits gelöst" war. Ein erklärender Hinweis hätte mir geholfen. Die Kritik an der Art und Weise wie Du mit mir hier umspringst halte ich aufrecht.